src/screens/tabs/Profile/screens/Contacts/index.js

```diff
@@ -24,8 +31,26 @@ const Contacts = ({ modalRef }) => {
   const addEditContactRef = useRef(null);
   const actionSheetRef = useRef(null);
   const [actionSheetData, setActionSheetData] = useState(undefined);
+  const { contacts, contactsLoading } = useSelector(state => state.user);
+  const dispatch = useDispatch();
 
-  const { groupedContacts, onDelete } = useContacts();
+  const groupedContacts = useMemo(
+    () =>
+      contacts.reduce((list, contact) => {
+        const { name } = contact;
+        const listItem = list.find(
+          item => item.letter && item.letter === getFirstLetterFrom(name)
+        );
+        if (!listItem) {
+          list.push({ letter: getFirstLetterFrom(name), contacts: [contact] });
+        } else {
+          listItem.contacts.push(contact);
+        }
+
+        return list;
```

Comment:
we could move this format function to a utils file or sth

Reply:
Agree